DiffAugment-stylegan2-pytorch/train.py

```diff
@@ -118,8 +118,7 @@ def setup_training_loop_kwargs(
             args.comet_api_key = comet_api_key
             experiment = comet_ml.Experiment(api_key=comet_api_key, project_name='Sirius SOTA GANs',
                                              auto_output_logging='simple', auto_log_co2=False,
-                                             auto_metric_logging=False, auto_param_logging=False,
-                                             auto_weight_logging=False)
```

Comment:
Why do we remove auto_weight_logging?

Reply:
`COMET WARNING: Config key 'comet.auto_log.weights' is deprecated, please use 'comet.auto_log.histogram_weights' instead`
And `auto_histogram_weight_logging` is `False` by default